src/webview/MessageList.js

```diff
@@ -25,6 +24,10 @@ import type {
   ThemeName,
   User,
 } from '../types';
+
+import type { ThemeColors } from '../styles';
+import { ThemeContext } from '../styles';
+
 import { connect } from '../react-redux';
```

Comment:
No blank lines between imports, except between the block of those from external libraries, at top, and from our own code, after that.

(I see this file already has a stray blank line, oops. But I think we're generally pretty consistent on this.)